.travis.yml

```diff
@@ -2,6 +2,11 @@ language: java
 jdk:
 - oraclejdk8
 dist: trusty
+addons:
```

Comment:
what does this do?

Reply:
This is for Selenium Web tests. 
We are testing our UI code against Chrome and Firefox, we need to install those two browsers at tun time.